.github/workflows/ci-angular.yaml

```diff
@@ -0,0 +1,80 @@
+name: Angular CI
+
+on:
+  push:
+    branches: [ "master" ]
+env:
+  IMAGE_NAME: angular
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+jobs:
+  testing:
+     runs-on: ubuntu-latest
+     outputs:
+        get-job-id-test: ${{steps.job-id-test.outputs.get-job-id-test-out}}
+        short-sha1: ${{steps.env-short-sha.outputs.short-sha}}
+     strategy:
+         fail-fast: true
+     steps:
+     - name: Take Short SHA
+       id: env-short-sha
+       run: echo "::set-output name=short-sha::${GITHUB_SHA::7}"
+     - uses: actions/checkout@v1
+     - name: Use Node.js
+       uses: actions/setup-node@v1
+       with:
+        node-version:  '18.6'
+     - name: Test angular
+       run: |
+        npm install
+        npm run test -- --watch=false --browsers=ChromeHeadless
+       working-directory: angular
+     - name: Get Current ID Job 
+       uses: Tiryoh/gha-jobid-action@v0
+       id: get-job-id-test
+       with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          job_name: ${{github.job}}
+          per_page: 30
+     - name: Set job_id testing
+       id: job-id-test
+       run: echo "::set-output name=get-job-id-test-out::${{steps.get-job-id-test.outputs.job_id}}"
+  
+  build_push:
```

Comment:
let's separate into 2
build_push should be broken into independent jobs.

Reply:
See how you can share artifacts between jobs.

Reply:
For share data between jobs I decided to use the cache, because this metod  is faster

Reply:
Fixed